lib/parser/livesplit_core_parser.rb

```diff
@@ -1,10 +1,10 @@
 begin
-  require File.expand_path('LiveSplitCore', __dir__)
+  require_relative('LiveSplitCore')
 rescue LoadError
-  raise 'Please follow the steps to install livesplit-core in the readme'
+  raise 'LiveSplitCore install not found'
 end
 
-class Parser
+class Parser::LivesplitCoreParser
```

Comment:
The new autoloader also did not like the way this was defined before.  Since `/lib` is the top level namespace, anything under it needs to be namespaced according to its directory structure.